ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/EventsTableFlatEventsRangeQueries.scala

```diff
@@ -108,11 +112,34 @@ private[events] sealed trait EventsTableFlatEventsRangeQueries[Offset] {
         }
       }
     }
+
+    frqK match {
+      case FrqK.ByArith(fasterRead, saferRead) =>
+        EventsRange.readPage(
+          fasterRead,
+          saferRead,
+          EventsTable.rawFlatEventParser,
+          offsetRange(offset),
+          pageSize)
+      case FrqK.ByLimit(sql) =>
+        SqlSequence.vector(sql withFetchSize Some(pageSize), EventsTable.rawFlatEventParser)
+    }
   }
 }
 
 private[events] object EventsTableFlatEventsRangeQueries {
 
+  private[EventsTableFlatEventsRangeQueries] sealed abstract class FrqK
```

Comment:
What does this stand for? I would prefer if you could write the full name, it would make it more readable.

Reply:
Er, "Flatevents Range Queries Continuation"? I'm not sure that clarifies anything though 😃 

Honestly I never thought about what it was called, I merely noticed that the return type of the various protected methods had nothing to do with the return type of the only public method, and was merely a protocol between the former and the latter, and therefore deserved an equally private shorthand. I'll...think of something.

Reply:
6c3c0e587631e5dedd7afa5cfce9a8201e2ae21c